examples/summary.zig

```diff
@@ -0,0 +1,40 @@
+const std = @import("std");
+const zbench = @import("zbench");
+const test_allocator = std.testing.allocator;
+
+fn fastFunction(_: std.mem.Allocator) void {
+    var result: usize = 0;
+    for (0..1000) |i| result += i * i;
+}
+fn mediumFunction(_: std.mem.Allocator) void {
+    var result: usize = 0;
+    for (0..20000) |i| result += i * i;
+}
+fn slowFunction(_: std.mem.Allocator) void {
+    var result: usize = 0;
+    for (0..300000) |i| result += i * i;
+}
```

Comment:
if you compile with options other than Debug, this will basically be optimized away, all functions will take the same time. The readme could contain a hint "compile in debug mode to try the examples".

Reply:
Oops, quite right. Would you put the `doNotOptimizeAway()` on `i`?